src/Pages/jongjin/Login/Login.js

```diff
@@ -1,13 +1,56 @@
-import React, { Component } from 'react';
-
-class Login extends Component {
-    render() {
-        return (
-            <div>
-                
-            </div>
-        );
-    }
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+import './Login.scss';
+
+class login extends React.Component {
+
+  state = {
+    id: "",
+    password: "",
+  }
+
+  handleInputChange = (e) => {
+    this.setState({
+      [e.target.name]: e.target.value
+    })
+  }
```

Comment:
```suggestion
  handleInputChange = (e) => {
    const { name, value } = e.target;
    
    this.setState({
      [name]: value
    })
  }
```
- 구조 분해 할당은 state, props 객체 뿐만 아니라 모든 객체, 배열에 적용할 수 있습니다.

Reply:
네! 구조분해 할당 최대한 활용해서 코드 가독성 높여보겠습니다~